lib/camera-scanner.js

```diff
@@ -96,20 +108,30 @@ export class CameraScanner extends EventEmitter {
   /**
   * Get formats array based on scan type.
   *
+  * If explicit formats provided via constructor, use those.
+  * Otherwise, use scanType defaults.
+  *
   * @private
   * @param {string} scanType - Scan type.
   * @returns {string[]} Array of format strings.
   */
   _getFormats(scanType) {
+    // Check for explicit format override first
+    if(this.config.formats !== null) {
+      return this.config.formats; // returns ['pdf417_enhanced']
```

Comment:
Put comments above to match our coding style... but does this comment still apply?

```suggestion
      // returns ['pdf417_enhanced']
      return this.config.formats;
```

Reply:
@dlongley 

Noted. Yes, this comment is still valid.

However, I have explained here ([link](https://github.com/digitalbazaar/bedrock-vue-optical-scanner/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R75-R82)) that the use is primarily to support legacy production application that uses the Dynamsoft SDK for pdf417 scan.

Addressed in this commit - https://github.com/digitalbazaar/bedrock-web-optical-scanner/pull/5/commits/dcfcb5cae4a7d8aa90bbdbe1a4ca62165d2fbbd9